components/Aduan/index.vue

```diff
@@ -1,9 +1,71 @@
 <template>
-  <div>Page Aduan Warga</div>
+  <div class="h-full flex justify-center sm:h-auto">
+    <div
+      class="bg-white w-full h-full p-4 flex flex-col justify-center items-center dark:bg-black sm:w-[360px] sm:h-[515px] sm:rounded-lg sm:dark:bg-dark-emphasis-low"
+    >
+      <div
+        class="flex justify-center items-center bg-gray-100 w-[120px] h-[120px] rounded-full dark:bg-dark-emphasis-medium m-auto"
+      >
+        <ImageLoudSpeaker class="" />
+      </div>
```

Comment:
there is an empty `class` here

Reply:
i am sorry, i forgot delete unused class [here](https://github.com/jabardigitalservice/super-app-web/pull/40/commits/a533b98d26a633c3278624c6a4489e4802aec88c)